ozkour-back/infrastructure/googledocs/googleDocRepository.js

```diff
@@ -0,0 +1,129 @@
+const wrapper = require('./docsWrapper')
+
+function removeTemplateText (documentId) {
+  const requests = [
+    {
+      deleteContentRange: {
+        range: {
+          startIndex: 1,
+          endIndex: 75
+        }
+      }
+    }
+  ]
+  return wrapper.updateDocument(documentId, requests)
+}
+
+function getSuccessMessage (documentId) {
+  return {
+    message: 'Created !',
+    link:
+        'https://docs.google.com/document/d/' +
+        documentId +
+        '/'
+  }
+}
+
+function addTitle (title, index) {
+  const end = index + (title.length)
+  const requests = [
+    {
+      insertText: {
+        text: title + '\n',
+        location: {
+          index
+        }
+      }
+    },
+    {
+      updateParagraphStyle: {
+        paragraphStyle: {
+          namedStyleType: 'HEADING_1'
+        },
+        fields: 'namedStyleType',
+        range: {
+          startIndex: index,
+          endIndex: end
+        }
+      }
+    }
+  ]
+  return requests
+}
+
+function addTalkInEmailing (index, titleTalk, speakers, date, eventName, url) {
+  const line = `${titleTalk}, animé par ${speakers}, le ${date} | ${eventName}` + '\n\n'
+  const end = index + (line.length)
+  const request = [
+    {
+      insertText: {
+        text: line,
+        location: {
+          index
+        }
+      }
+    },
+    {
+      updateParagraphStyle: {
+        paragraphStyle: {
+          namedStyleType: 'NORMAL_TEXT'
+        },
+        fields: 'namedStyleType',
+        range: {
+          startIndex: index,
+          endIndex: end
+        }
+      }
+    }
+  ]
+
+  if (url) {
+    request.push({
+      updateTextStyle: {
+        textStyle: {
+          link: {
+            url
+          }
+        },
+        fields: 'link',
+        range: {
+          startIndex: index,
+          endIndex: index + titleTalk.length
+        }
+      }
+    })
+  }
+  return request
+}
+
+async function addTextForEmailing (documentId, mapUniverse) {
+  const requests = []
+
+  const mapIter = mapUniverse.keys()
+  let universe = mapIter.next().value
+  let index = 1
+  while (universe) {
```

Comment:
Pas besoin d'utiliser un iterateur  
[forEach((value, key, map) => { /* … */ } )](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/forEach)
```suggestion
  let index = 1;
  mapUniverse.forEach((universe) => {
```